### PR TITLE
update lesson for postgres

### PR DESCRIPTION
**Pull request by linda-seiter**

### CODE REVIEW

**Review by developerjaym — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -2,316 +2,569 @@
...
+We will continue with the `Student` entity from the previous lessons.
 
 ## Introduction
 
 Relational database tables can be built such that they associate with one
 another via primary and foreign keys. JPA provides us with annotations that
 streamline model association in our program.
 
-In this lesson, we will be building out the following relationship:
+In this lesson, we will use implement the following relationship:
```
Comment (developerjaym) [inline README.md]: Consider removing 'use'

Comment (linda-seiter) [reply README.md]: fixed, thanks

File: README.md [hunk 2]
```diff
@@ -388,71 +647,427 @@ Open the `Student` class and add the `(fetch = FetchType.LAZY)` property to the
...
+
+   //getters/setters/toString
 ```
 
-A one-to-one association uses the `FetchType.EAGER` value if no value is
-specified. By setting it to `FetchType.LAZY`, we can defer the join query. Run
-the `main` method of the `JpaRead` class again to see how the Hibernate query
+A one-to-one association uses the `FetchType.EAGER` value as a default if no value is
+specified. By setting it to `FetchType.LAZY`, we can defer the join query
+until the `getCard()` method is called.
```
Comment (kcleland0818) [inline README.md:677]: I think I might be reviewing these PRs out of order; if this lesson comes before the lesson about removing rows in a cascade fashion, ignore that comment on that lesson as I see the fetch is covered here.

**Review by kcleland0818 — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (linda-seiter) [reply README.md:677]: ok thanks

**Review by developerjaym — APPROVED** [review]

**Review by AlveeM — APPROVED** [review]

**Pull request merged**